libsolidity/lsp/LanguageServer.cpp

```diff
@@ -101,14 +101,29 @@ Json::Value LanguageServer::toJson(SourceLocation const& _location)
 void LanguageServer::changeConfiguration(Json::Value const& _settings)
 {
 	m_settingsObject = _settings;
+	Json::Value jsonIncludePaths = _settings["include-paths"];
+	if (jsonIncludePaths && jsonIncludePaths.isArray())
```

Comment:
Not sure if I missed it earlier, or if it changed, but in the current code `jsonIncludePaths.isArray() == false` does not result in an `Invalid JSON configuration passed.` report.

Reply:
I'm fine with being strict, just not sure if the LSP clients are actually that strict themselves. And if they aren't, we'll have to adapt to them not they to us. I can see them just not being consistent and deciding to set a field to `null` instead of removing it in some cases. We had been doing that ourselves in JSON exports for a long time.

Anyway, I also think this is minor. Just wanted to let you know that this could potentially lead to a false-positive warning, depending on what clients do.